### DISCUSSION TITLE
problems drawing short lines

### DISCUSSION THREAD

**Discussion by SanelyInproductive** [xournalpp/xournalpp#2818] [Q&A]
Hello, 
I am using Xournal++ on Windows with a Wacom IntuosS. Unfortunately, when I try to write a text, I have trouble writing commas or the point of the letter i (usually I get a circle around the cursor then).
![image](https://user-images.githubusercontent.com/79225950/108262937-df7bcc00-7165-11eb-998c-b4d84d0438bc.png)

This leads me to results like this
![image](https://user-images.githubusercontent.com/79225950/108263291-5913ba00-7166-11eb-907b-18b26153d447.png)
which is supposed to say "Hello, I am trying to write a text here", but sometimes these circles mess up my handwriting
Is there some sort of button that I should be pressing to fix this? (With other softwares I do not have this problem)

Thanks in advance!

**Comment by personalizedrefrigerator** [comment]
This looks like a bug! Unfortunately, I don't have a Windows testing environment and can't reproduce this on Linux.

If this happens with [v1.1.0+dev](https://github.com/xournalpp/xournalpp/releases/tag/nightly) and the new input system enabled, please [file a bug report](https://github.com/xournalpp/xournalpp/issues/new?assignees=&labels=Input%2C+bug&template=input-related-bug-report.md&title=) as well!
